_plugins/lib.rb

```diff
@@ -16,24 +16,52 @@ def gen_values(versions, imageNames, imageRegistry)
     ipam: calico-ipam
     app_layer_policy: false
 
+    prodname: Calico
```

Comment:
Do we need these two?

Reply:
Those are used in the 3.6 templates and nowhere else. It seems like a divergence between the 3.6 templates and the following manifests. I need to take a look at it later to reconcile which one is correct but for now, this will make it so all of them function properly.